src/pages/Index/Section/NftInfluencers/accounts.json

```diff
@@ -0,0 +1,475 @@
+[
+    {
+      "Name": "Pranksy",
+      "Address": "0xd387a6e4e84a6c86bd90c158c6028a58cc8ac459",
+      "ENS": "",
+      "Twitter": "https://twitter.com/Pranksy",
+      "OpenSea": "https://opensea.io/accounts/Pranksy",
+      "Youtube": "",
+      "Notes": "Proof - https://dune.xyz/queries/40614/80127\n&& OpenSea",
+      "Location": "",
+      "Achievment": ""
```

Comment:
Can it be fetched by using api?

Reply:
It is not ready, https://discord.com/channels/334289660698427392/483922119873265664/921417515744264262

Reply:
True. Can't be yet